### PR TITLE
fixes #84, fixes #75, fixes #51, fixes #42 - fixes code chun, broke guid...

### LINKED ISSUE #42: There should not be a space between the dash and the declaration of "some_array"s
In the variables section of the Lavender guide, there should not be a space between the dash and the declaration of "some_array"s. -some_array works in code as does - some_array.

### LINKED ISSUE #51: The significance of white space should be emphasized in the language guide.
The significance of white space should be emphasized. 

Line 86 in views/layouts/default.lavender the last bit of code is this:

This (space between parentheses and operator) does not work:

 li(class=className)
              a(
              href=urlPage[index],
              title=titles[index],
              target=target[index],
                ) =titles[index]

 This (no space between parentheses and operator) does work:

li(class=className)
              a(
              href=urlPage[index],
              title=titles[index],
              target=target[index],
                )=titles[index]

Feature or bug?

variable text should be documented

whitespace explanation

In the code example above, the white space did not add a new line, it just made the =titles[index] array reference fail to work.

### LINKED ISSUE #75: "=" symbol should be rendered with a standard of no space

### LINKED ISSUE #84: Code chunks should not break up over two lines
don t let code bars break up

### PR DESCRIPTION
**Pull request by sugarPhlox**
fixes #84 ,
fixes #75 ,
fixes #51 ,
fixes #42
Fixes broken up code chunks, also fixes the floating nav bar (would disappear behind guide when switching from desktop to mobile). Fixes some other things involving spaces in the guide. Fix fix fix.

### CODE REVIEW

File: views/guide.lavender [hunk 1]
```diff
@@ -170,12 +171,15 @@ block content
...
+            | without outputting anything to the page.
           p
-            | This just assigns a variable without outputting anything to the page.
+            | Note: While we do write it with no space, it is still valid with and extra space as
+            code.language-markup - some_variable
+            | . Expressions in general are not whitespace sensitive.
             pre 
               code.language-jade.
-               - some_variable = 'this is stored in a variable'
+               -some_variable= 'this is stored in a variable'
```
Comment (TomWoodward) [inline views/guide.lavender]: you should have a space before this `=`. theres a difference between `p= foobar` and `-variable = 'foobar'`

**Comment by TomWoodward** [comment]
looks like there was some confusion in what i said in the issues about whitespace regarding the `=` sign, definitely reach out to me if you're not getting what i'm saying.

**Comment by sugarPhlox** [comment]
Dan explained it, it makes a bit more sense for me. It was also correct in the lavender wiki, so I just looked back on that to help.

**Comment by mowchan** [comment]
:cake:

File: views/guide.lavender [hunk 2]
```diff
@@ -438,7 +444,7 @@ block content
           | (or will be) plenty in place out of the box. Here is how they work!
           pre
             code.language-jade.
-              - myvariable = 'some really cool text'
+              - myvariable= 'some really cool text'
```
Comment (zlincoln) [inline views/guide.lavender]: missed this one I believe

**Comment by zlincoln** [comment]
:cake:

**Pull request merged**